features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/custom-authentication/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "custom-authentication",
+  "name": "Custom Authentication",
+  "description": "Enable login for users with external authentication service.",
+  "image": "",
+  "category": "DEFAULT",
```

Comment:
What is meant by this param. What are the other available categories

Reply:
Currently, all the available connections fall under the DEFAULT category. If we are introducing a new category with custom authentication, we would have to initiate that separately.